api/src/test/java/org/pmiops/workbench/api/CohortBuilderControllerTest.java

```diff
@@ -30,7 +31,13 @@
 
     @Override
     public List<String> getTableNames() {
-        return Arrays.asList("icd9_criteria", "person", "CONCEPT", "condition_occurrence");
+        return Arrays.asList(
+                "icd9_criteria",
+                "person",
+                "CONCEPT",
```

Comment:
Is there a reason this one is in all caps?

Reply:
BigQuery table names are case sensitive, ref:
https://cloud.google.com/bigquery/docs/reference/standard-sql/lexical#case-sensitivity.  To make this lowercase we'll have to adjust the table schema in BQ.

Reply:
Yeah.. table names are case sensitive in BigQuery.

Reply:
OK. Let's file an issue to change the table names in BQ to be all lower-case?

Reply:
Yep.. should that be posted in the RW project in jira or somewhere esle? And who should I assign it to?

Reply:
RW project is fine. You can assign it to me if you want.